Update README.md (#406)
w3f/General-Grants-Program: README.md

Additionally, it must fulfill the following requirements:
- All code produced as part of a grant must be **open-sourced**, and it must also not rely on closed-source software for full funcionality. We prefer Apache 2.0 but the GNU GPL v3 license or Unlicense are also acceptable. 

> - We do not award grants to teams that have successfully conducted a token sale for the purpose of the grant itself. 

Reviewer: Oh I see. I had misunderstood it myself. Maybe this 

```suggestion
- We do not award grants for projects that have been the object of a successful token sale.
```

> - Lastly, we do not provide funding for gambling-, drug or gun-related projects.

Reviewer: ```suggestion
- Lastly, we do not fund projects that actively encourage gambling, illicit trade, money laundering or criminal activities in general.
```
